internal/utils/labelsutils.go

```diff
@@ -3,20 +3,10 @@ package utils
 import (
 	"fmt"
 
-	v1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/labels"
 	"k8s.io/apimachinery/pkg/selection"
 )
 
-func IsNodeSchedulable(node *v1.Node) bool {
-	for _, taint := range node.Spec.Taints {
-		if taint.Effect == v1.TaintEffectNoSchedule {
-			return false
-		}
-	}
-	return true
-}
-
 func IsObjectSelectedByLabels(objectLabels map[string]string, selectorLabels map[string]string) (bool, error) {
```

Comment:
Can we move this function to `internal/utils/kmmlabels.go` and completely remove this file?

Reply:
@ybettan done